src/Synapse/Synapse/Common/HelpMessages.cs

```diff
@@ -165,5 +165,17 @@ public static class HelpMessages
         public const string EndIpAddress = "The end IP address of the firewall rule. Must be IPv4 format. Must be greater than or equal to startIpAddress.";
 
         public const string AzureIpRule = "Creates a special firewall rule that permits all Azure IPs to have access.";
+
+        public const string RoleAssignmentId = "The ID of the role assignment.";
+
+        public const string RoleDefinitionId = "Id of the Role that is assigned to the principal.";
+
+        public const string RoleDefinitionName = "Name of the Role that is assigned to the principal.";
+
+        public const string PrincipalId = "The Azure AD ObjectId of the User, Group or Service Principal.";
+
+        public const string SignInName = "The email address or the user principal name of the user.";
+
+        public const string ServicePrincipalName = "The ServicePrincipalName of the service principal.";
```

Comment:
>"The ServicePrincipalName of the service principal. [](start = 51, length = 51)

@wangdongwei@pku.edu.cn  Do you prefer "The name of the service principal."?

Reply:
Actually "the name of the sp" is confused. because despite `ServicePrincipalName`, you also have `DisplayName`. You can use `Get-AzADServicePrincipal` to check the difference.

Reply:
> Actually "the name of the sp" is confused. because despite `ServicePrincipalName`, you also have `DisplayName`. You can use `Get-AzADServicePrincipal` to check the difference.

Got it. Thanks for helping me learn more.